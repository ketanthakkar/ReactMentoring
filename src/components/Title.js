import React from 'react';

const Title = () => (
      <div className="title-section">
        <h4 className="netflix-text">netflixroulette</h4>
      </div>
)
  
export default Title;