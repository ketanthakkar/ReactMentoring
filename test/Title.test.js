import React from 'react';
import { mount } from 'enzyme';
import Title from "../src/components/Title";

describe('should renders Title correctly', () => {
  const tree = mount(<Title />);
  test('Snapshot test', () => {
    expect(tree).toMatchSnapshot();
  });
});