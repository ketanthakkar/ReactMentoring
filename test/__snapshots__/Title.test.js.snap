// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should renders Title correctly Snapshot test 1`] = `
<Title>
  <div
    className="title-section"
  >
    <h4
      className="netflix-text"
    >
      netflixroulette
    </h4>
  </div>
</Title>
`;
